src/components/Orders/OrdersHistoryItem.tsx

```diff
@@ -41,7 +45,7 @@ const OrdersHistoryItem = () => {
 
         {data && data.data.length > 0 && (
           <>
-            {data.data.map((el) => {
+            {data.data.map((el: IOrdersHistoryItem) => {
```

Comment:
The typing is unnecessary as the generic provided to useQuery should have the correct type already